package.json

```diff
@@ -57,9 +58,11 @@
     "eslint": "^7.26.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-jest": "^24.3.6",
+    "genversion": "^3.1.1",
     "jest": "^27.0.6",
     "jest-watch-typeahead": "^0.6.4",
     "prettier": "^2.3.2",
+    "secretsfoundry": "^0.1.16",
```

Comment:
This I assume needs to be removed?
Although why does it get added in the first place :/